test/test_icalparser.py

```diff
@@ -102,4 +102,8 @@ def test_attendee(self):
     def test_organizer(self):
         self.assertIsInstance(self.eventA.organizer, str)
         self.assertIsInstance(self.eventB.organizer, str)
-        
\ No newline at end of file
+        
+    def test_str(self):
+        self.eventA.start = datetime(year=2017, month=2, day=3, hour=12, minute=5)
+        self.eventA.end = datetime(year=2017, month=2, day=3, hour=15, minute=5)
+        self.assertEqual('2017-02-03 12:05:00+00:00: Event A (ended)', str(self.eventA))
```

Comment:
I can't see how this actually tests your proposed change. You test direct comparison, just the string representations.

Reply:
The bug reported in #69 is that `str(event)` fails for all-day events, which use naive datetime objects. This tests `str(event)` which is where the patch is and the bug occurs. The test added here fails in master and  no longer fails with this patch. The error was introduced by [#66](https://github.com/irgangla/icalevents/pull/66/files#diff-6659079db1c5e26c4bd075f64b432522644ff3ee3bd4264f2863bd6f69d63e72R303-R315) which enforces that all_day events do not have any timezone associated with them and use naive datetimes, so no code using event.start/end may assume tz-aware datetime objects for comparison.

Reply:
Oh, I see now. 
For some reason I missed that this was in the __str__-method when I reviewed it the first time.

Reply:
Thanks for the pointer in the right direction @minrk. 

I’ve approved and merged this now, I can unfortunately not release  new versions to pypi as I don’t have the credentials so we’ll have to wait for @irgangla to get this released.